backend/action_logs/migrations/0008_rename_parent_actionlogcomment_parent_comment.py

```
@@ -0,0 +1,18 @@
+
+    dependencies = [
+        ('action_logs', '0007_alter_actionlogcomment_options_and_more'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='actionlogcomment',
+            old_name='parent',
+            new_name='parent_comment',
+        ),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)